src/mame/motorola/mvme147.cpp

```diff
@@ -161,21 +161,17 @@
 #include "machine/clock.h"
 #include "machine/timekpr.h"
 
-#define LOG_GENERAL 0x01
-#define LOG_SETUP   0x02
-#define LOG_PRINTF  0x04
+#define LOG_SETUP   (1U << 1)
+#define LOG_PRINTF  (0)
 
-#define VERBOSE 0 // (LOG_PRINTF | LOG_SETUP  | LOG_GENERAL)
+#define VERBOSE (0) // (LOG_SETUP | LOG_GENERAL)
 
-#define LOGMASK(mask, ...)   do { if (VERBOSE & mask) logerror(__VA_ARGS__); } while (0)
-#define LOGLEVEL(mask, level, ...) do { if ((VERBOSE & mask) >= level) logerror(__VA_ARGS__); } while (0)
+#if VERBOSE && LOG_PRINTF
+#define LOG_OUTPUT_FUNC printf
+#endif
 
-#define LOG(...)      LOGMASK(LOG_GENERAL, __VA_ARGS__)
-#define LOGSETUP(...) LOGMASK(LOG_SETUP,   __VA_ARGS__)
+#include "logmacro.h"
 
-#if VERBOSE & LOG_PRINTF
-#define logerror printf
-#endif
```

Comment:
This is like the stuff in src/devices/bus/vme – get rid of `LOG_PRINTF` altogether and provide a commented `#define LOG_OUTPUT_FUNC osd_printf_info`.

Reply:
Resolved as in previous VME files.